lib/modules/platform/gitlab/index.ts

```diff
@@ -136,6 +137,7 @@ export async function initPlatform({
       platformConfig.gitAuthor = `${user.name} <${
         user.commit_email ?? user.email
       }>`;
+      config.author = user.name;
```

Comment:
we should probably allow using the manual configured username if we can't determine.
can be done in a followup pr, gitea doesn't fallback either.